### ISSUE TITLE
which targets were rejected on import?

### ISSUE THREAD

**Issue by digininja** [Raikia/FiercePhish#73]
I had 3 targets in the system, I imported a CSV containing 562, I now have 563 targets in the system. The import failed for 2 of the entries but I don't know which ones. You need some way to show which lines failed.

**Comment by Raikia** [comment]
Yeah, thats true.  The import job is a background job so I didn't make any type of error notification system for it, which was shortsighted.  I'll make sure I address this in the next iteration!  Good call

**Comment by digininja** [comment]
For anyone interested, the way I worked it out was doing this:

```
select target_users.first_name, target_users.last_name, target_users.email from target_users into outfile "/var/lib/mysql-files/imported.csv" FIELDS TERMINATED BY ',' ENCLOSED BY '"' LINES TERMINATED BY '\n';
```

and then sorting and diffing the file with my input file.
